src/Microsoft.ML.Data/Transforms/CopyColumnsTransform.cs

```diff
@@ -130,18 +109,21 @@ public sealed class Arguments : TransformInputBase
             public Column[] Column;
         }
 
-        // Factory method for SignatureLoadDataTransform.
-        private static IDataView Create(IHostEnvironment env, ModelLoadContext ctx, IDataView input)
+        /// <summary>
+        /// Factory method to create from arguments
+        /// </summary>
+        public static IDataTransform Create(IHostEnvironment env, Arguments args, IDataView input)
```

Comment:
>public [](start = 8, length = 6)

you can make it private. and if it becomes private, you don't have to change comments to //summary, and leave them as before.